src/main/java/io/vertx/ext/hawkular/VertxHawkularOptions.java

```diff
@@ -356,4 +362,19 @@ public VertxHawkularOptions setMetricsBridgeEnabled(boolean metricsBridgeEnabled
     this.metricsBridgeEnabled = metricsBridgeEnabled;
     return this;
   }
+  /**
+   * Set metric that will not be registered. Schedulers will check the set {@code disabledMetricsTypes} when
+   * registering metrics suppliers
+   *
+   * @param metricsType the type of metrics
+   * @return the current {@link VertxHawkularOptions} instance
+   */
+  public VertxHawkularOptions setDisabledMetricsType(MetricsType metricsType) {
```

Comment:
This is not a setter. This is a convenience method. If you want to keep it, please name it `#addDisabledMetricsType`